README.md

```diff
@@ -1,2 +1,17 @@
-# echo-heaps
-simple example for integrating [echo physics](https://austineast.dev/echo/) with the [heaps engine](https://heaps.io)
+# What is this
+
+- This is a game that integrating [echo physics](https://austineast.dev/echo/) with the [heaps engine](https://heaps.io)
+
+## How to start
+```
+# Install deps from both hashlink and web and the shared libs
+$ haxelib install all 
+
+# To produce hashlink target
+$ haxe hl.hxml
+
+# To produce browser target
+$ haxe js.hxml 
+```
+
+#### This can also be configured to produce targets for android too
```

Comment:
And finally, I think this line should be removed. I personally havent gotten heaps running on android, so i dont want to mention something thats not easy to set up in this repo 😅

Reply:
aah damn me too, i've actually be playing on the heaps android repo and im being unable to compile. i wish there was just ONE working sample online on github

Reply:
this one might be working on windows, they are still trying to get the docs and samples for windows and linux in. https://github.com/HeapsIO/heaps-android/pull/2

Reply:
i've removed the text, would you be interested in trying to get a sample working?

Reply:
thanks! Its definitely something im interested in, but unfortunately im not very familiar with android development.